src/main/java/org/jenkinsci/plugins/Beer.java

```diff
@@ -21,6 +23,11 @@ public String getImage() {
     }
     
     public String getIconFileName() {
+        // Custom symbols are only available in 2.341 or above
+        if (new VersionNumber(Jenkins.VERSION).isNewerThanOrEqualTo(new VersionNumber("2.341"))) {
+            return "symbol-beer plugin-beer";
+        }
```

Comment:
I'd vouch for simply bumping the baseline, otherwise we have this change available on the current LTS line, which does still use tango icons and would look out of place there.
This is more something for the upcoming weekly instance to show off.

Reply:
Updated it - hope I did it the right way

Reply:
Looks alright, but we can just 4.40 straight away.